Xamarin.Forms.Platform.UAP/BoxViewRenderer2.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.ComponentModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Shapes;
+using Xamarin.Forms.Platform.UWP;
+
+namespace Xamarin.Forms.Platform.UAP
+{
+	class BoxViewRenderer2 : ViewRenderer<BoxView, Border>
```

Comment:
Needs to be public so people can write custom renderers. Also, not really a big fan of this name.

Reply:
Thanks, I'll fix that.

Reply:
Any suggestion on the name?  I'm struggling there also.  Trying to think of a meaningful name.

Reply:
I'll go with BoxViewBorderRenderer for now